contrib/extractor.bat

```diff
@@ -1,57 +1,76 @@
 @ECHO OFF
-CLS
-
-:MENU
-ECHO.
-ECHO ...............................................
-ECHO Trinitycore dbc/db2, maps, vmaps, mmaps extractor
-ECHO ...............................................
-ECHO PRESS 1, 2, 3 OR 4 to select your task, or 5 to EXIT.
-ECHO ...............................................
-ECHO.
-ECHO The vmaps extractor output text below is intended and not an error:
-ECHO ..........................................
-ECHO Extracting World\Wmo\Band\Final_Stage.wmo
-ECHO No such file.
-ECHO Couldn't open RootWmo!!!
-ECHO Done!
-ECHO ..........................................
-ECHO.
-ECHO 1 - Extract dbc/db2 and maps
-ECHO 2 - Extract vmaps (needs maps to be extracted before you run this)
-ECHO 3 - Extract mmaps (needs vmaps to be extracted before you run this, may take hours)
-ECHO 4 - Extract all (may take hours)
-ECHO 5 - EXIT
-ECHO.
-SET /P M=Type 1, 2, 3, 4 or 5 then press ENTER:
-IF %M%==1 GOTO MAPS
-IF %M%==2 GOTO VMAPS
-IF %M%==3 GOTO MMAPS
```

Comment:
Add these features back for people who don't want to extract mmaps every time they use this batch script.
My slow 2012 computer uses more than 6 hours to extract those for WotLK, 8-12 hours for Warlords/Legion.